### PR TITLE
splinesans: v1.000 added

### PR DESCRIPTION
**Pull request by bghryct**
Taken from the upstream repo https://github.com/SorkinType/SplineSans at commit https://github.com/SorkinType/SplineSans/commit/2c722946af3cfa5fde6c2ea1c0813ade7fcb283a

### CODE REVIEW

File: ofl/splinesans/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<p>Spline Sans is a Grotesque purpose-built for UI interfaces, checkout processes, and paragraphs of text. Space efficiency is accomplished by condensing traditional grotesque proportions. The cool and restrained tone is accented with strategic "thorn" traps, which blossom into view when set at larger sizes.</p> 
+<p>Spline Sans is an original typeface initiated by the <a href="https://spline.design/">Spline Team</a> and designed by Eben Sorkin and Mirko Velimirovic.
+Project manager - Faride Mereb. Testing and design feedback - Gonçalo Teixeira. Concept - Alejandro León.</p>
+<p>Spline Sans supports Google Core, Google Plus, Google Pro, Plus Optional, and Pro optional glyph sets, enabling the typesetting of English, and Western European languages.</p>
```
Comment (RosaWagner) [inline ofl/splinesans/DESCRIPTION.en_us.html:4]: The names of the google fonts glyphsets are probably going to change in the future and don't make much sense to users

Comment (bghryct) [reply ofl/splinesans/DESCRIPTION.en_us.html:4]: That's all good. I think Eben is going to push a small amount of kerning fixes as well in the next week.

Comment (RosaWagner) [reply ofl/splinesans/DESCRIPTION.en_us.html:4]: Yes, I am waiting for the next meeting so we improve the entire build etc ;) happy thanksgiving

Comment (davelab6) [reply ofl/splinesans/DESCRIPTION.en_us.html]: I fixed the description file

**Comment by bghryct** [comment]
I've updated the splinesans to include Eben's kerning. @RosaWagner I'm unclear on the CLA issue, should I close this PR and open a new one with the finished work?

**Comment by emmamarichal** [comment]
## **QA :**

[SplineSans_qa.zip](https://github.com/google/fonts/files/7674903/SplineSans_qa.zip)

## Fontbakery report

Fontbakery version: 0.8.4

<details>
<summary><b>[8] SplineSans[wght].ttf</b></summary>
<details>
<summary>⚠ <b>WARN:</b> Checking OS/2 achVendID.</summary>

* [com.google.fonts/check/vendor_id](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/vendor_id)
<pre>--- Rationale ---
Microsoft keeps a list of font vendors and their respective contact info. This
list is updated regularly and is indexed by a 4-char &quot;Vendor ID&quot; which is stored
in the achVendID field of the OS/2 table.
Registering your ID is not mandatory, but it is a good practice since some
applications may display the type designer / type foundry contact info on some
dialog and also because that info will be visible on Microsoft&#x27;s website:
https://docs.microsoft.com/en-us/typography/vendors/
This check verifies whether or not a given font&#x27;s vendor ID is registered in
that list or if it has some of the default values used by the most common font
editors.
Each new FontBakery release includes a cached copy of that list of vendor IDs.
If you registered recently, you&#x27;re safe to ignore warnings emitted by this
check, since your ID will soon be included in one of our upcoming releases.</pre>

* ⚠ **WARN** OS/2 VendorID value 'NONE' is not yet recognized. If you registered it recently, then it's safe to ignore this warning message. Otherwise, you should set it to your own unique 4 character code, and register it with Microsoft at https://www.microsoft.com/typography/links/vendorlist.aspx
 [code: unknown]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
<pre>--- Rationale ---
Fonts with ligatures should have kerning on the corresponding non-ligated
sequences for text where ligatures aren&#x27;t used (eg
https://github.com/impallari/Raleway/issues/14).</pre>

* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + i
	- i + l

   [code: lacks-kern-info]

</details>
<details>
<summary>⚠ <b>WARN:</b> A static fonts directory with at least two fonts must accompany variable fonts</summary>

* [com.google.fonts/check/repo/vf_has_static_fonts](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/repo/vf_has_static_fonts)
<pre>--- Rationale ---
Variable font family directories kept in the google/fonts git repo may include a
static/ subdir containing static fonts.
These files are meant to be served for users that still lack support for
variable fonts in their web browsers.</pre>

* ⚠ **WARN** Please consider adding a subdirectory called "static/" and including in it static font files. [code: missing]

</details>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Designers are listed correctly on the Google Fonts catalog?</summary>

* [com.google.fonts/check/metadata/designer_profiles](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/designer_profiles)
<pre>--- Rationale ---
Google Fonts has a catalog of designers.
This check ensures that the online entries of the catalog can be found based on
the designer names listed on the METADATA.pb file.
It also validates the URLs and file formats are all correctly set.</pre>

* ⚠ **WARN** It seems that Mirko Velimirovic is still not listed on the designers catalog. Please submit a photo and a link to a webpage where people can learn more about the work of this designer/typefoundry. [code: profile-not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Ensure Stylistic Sets have description.</summary>

* [com.google.fonts/check/stylisticset_description](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/stylisticset_description)
<pre>--- Rationale ---
Stylistic sets should provide description text. Programs such as InDesign,
TextEdit and Inkscape use that info to display to the users so that they know
what a given stylistic set offers.</pre>

* ⚠ **WARN** The stylistic set ss01 lacks a description string on the 'name' table. [code: missing-description]
* ⚠ **WARN** The stylistic set ss02 lacks a description string on the 'name' table. [code: missing-description]
* ⚠ **WARN** The stylistic set ss03 lacks a description string on the 'name' table. [code: missing-description]

</details>
<details>
<summary>⚠ <b>WARN:</b> Ensure fonts have ScriptLangTags declared on the 'meta' table.</summary>

* [com.google.fonts/check/meta/script_lang_tags](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/meta/script_lang_tags)
<pre>--- Rationale ---
The OpenType &#x27;meta&#x27; table originated at Apple. Microsoft added it to OT with
just two DataMap records:
- dlng: comma-separated ScriptLangTags that indicate which scripts, or languages
and scripts, with possible variants, the font is designed for
- slng: comma-separated ScriptLangTags that indicate which scripts, or languages
and scripts, with possible variants, the font supports
The slng structure is intended to describe which languages and scripts the font
overall supports. For example, a Traditional Chinese font that also contains
Latin characters, can indicate Hant,Latn, showing that it supports Hant, the
Traditional Chinese variant of the Hani script, and it also supports the Latn
script
The dlng structure is far more interesting. A font may contain various glyphs,
but only a particular subset of the glyphs may be truly &quot;leading&quot; in the design,
while other glyphs may have been included for technical reasons. Such a
Traditional Chinese font could only list Hant there, showing that it’s designed
for Traditional Chinese, but the font would omit Latn, because the developers
don’t think the font is really recommended for purely Latin-script use.
The tags used in the structures can comprise just script, or also language and
script. For example, if a font has Bulgarian Cyrillic alternates in the locl
feature for the cyrl BGR OT languagesystem, it could also indicate in dlng
explicitly that it supports bul-Cyrl. (Note that the scripts and languages in
meta use the ISO language and script codes, not the OpenType ones).
This check ensures that the font has the meta table containing the slng and dlng
structures.
All families in the Google Fonts collection should contain the &#x27;meta&#x27; table.
Windows 10 already uses it when deciding on which fonts to fall back to. The
Google Fonts API and also other environments could use the data for smarter
filtering. Most importantly, those entries should be added to the Noto fonts.
In the font making process, some environments store this data in external files
already. But the meta table provides a convenient way to store this inside the
font file, so some tools may add the data, and unrelated tools may read this
data. This makes the solution much more portable and universal.</pre>

* ⚠ **WARN** This font file does not have a 'meta' table. [code: lacks-meta-table]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check font contains no unreachable glyphs</summary>

* [com.google.fonts/check/unreachable_glyphs](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/universal.html#com.google.fonts/check/unreachable_glyphs)
<pre>--- Rationale ---
Glyphs are either accessible directly through Unicode codepoints or through
substitution rules. Any glyphs not accessible by either of these means are
redundant and serve only to increase the font&#x27;s file size.</pre>

* ⚠ **WARN** The following glyphs could not be reached by codepoint or substitution rules:
 - slashl_part.
 - macronbelow
 - slashL_part.
 - ringacutecomb
 - .null
 - uni030C.alt.001
 - uni0326.002
 - dotbelow
 - g.ss02.alt
 - oSlash_part.
 - Barmid_part.
 - uni0308.001
 - ij_acutecomb
 - Bar_part.
 - ringcomba_part.
 - uni030C.alt
 - IJ_acutecomb
 - OSlash_part.
 - commaturnedabove 
 - circumflexbelow
 [code: unreachable-glyphs]

</details>
<details>
<summary>⚠ <b>WARN:</b> Are there any misaligned on-curve points?</summary>

* [com.google.fonts/check/outline_alignment_miss](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_alignment_miss)
<pre>--- Rationale ---
This check heuristically looks for on-curve points which are close to, but do
not sit on, significant boundary coordinates. For example, a point which has a
Y-coordinate of 1 or -1 might be a misplaced baseline point. As well as the
baseline, here we also check for points near the x-height (but only for lower
case Latin letters), cap-height, ascender and descender Y coordinates.
Not all such misaligned curve points are a mistake, and sometimes the design may
call for points in locations near the boundaries. As this check is liable to
generate significant numbers of false positives, it will pass if there are more
than 100 reported misalignments.</pre>

* ⚠ **WARN** The following glyphs have on-curve points which have potentially incorrect y coordinates:
	* dollar (U+0024): X=587.0,Y=1.0 (should be at baseline 0?)
	* dollar (U+0024): X=587.0,Y=1.0 (should be at baseline 0?)
	* ampersand (U+0026): X=1336.0,Y=-1.0 (should be at baseline 0?)
	* ampersand (U+0026): X=1116.0,Y=-1.0 (should be at baseline 0?)
	* nine (U+0039): X=358.5,Y=1452.5 (should be at cap-height 1454?)
	* at (U+0040): X=1179.5,Y=-2.0 (should be at baseline 0?)
	* U (U+0055): X=139.0,Y=1456.0 (should be at cap-height 1454?)
	* U (U+0055): X=330.0,Y=1456.0 (should be at cap-height 1454?)
	* f (U+0066): X=59.0,Y=1092.0 (should be at x-height 1091?)
	* f (U+0066): X=676.0,Y=1092.0 (should be at x-height 1091?) and 83 more. [code: found-misalignments]

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 0 | 8 | 53 | 9 | 150 | 0 |
| 0% | 0% | 4% | 24% | 4% | 68% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **INFO**
* **PASS**
* **DEBUG**

**Pull request merged**